apps/editors/helpers.py

```diff
@@ -750,12 +750,12 @@ def process_public(self):
         # Hold onto the status before we change it.
         status = self.addon.status
 
-        # Sign addon.
-        self.version.sign_files()
-
         self.set_files(amo.STATUS_PUBLIC, self.data['addon_files'],
                        copy_to_mirror=True)
 
+        # Sign addon.
+        self.version.sign_files()
```

Comment:
Why this change? Is there a chance someone could get the file before it is signed?